charts/ceph-csi-cephfs/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 apiVersion: v1
-appVersion: canary
+appVersion: v3.4.0
 description: "Container Storage Interface (CSI) driver,
 provisioner, snapshotter and attacher for Ceph cephfs"
 name: ceph-csi-cephfs
-version: 1.3.0-canary
+version: 3.4.0-canary
```

Comment:
this still includes `-canary`?

Reply:
@nixpanic yeah, we keep this https://github.com/ceph/ceph-csi/pull/2046/files#diff-f1b11c5f81f992e01a7019d45b7fc930ca4f6268b1e7ff018ee5a327950b4e2d